### DISCUSSION TITLE
完美校园适配和点击悬浮窗快照截取没反应

### DISCUSSION THREAD

**Discussion by yybcx** [gkd-kit/gkd#116] [General]
1.完美校园跳更新和进入以后的更改密码提示
2.本来想提交订阅的，但是截屏服务用不了，授权都给了但是就是没法截取快照，求解决

**Comment by yybcx** [comment]
三星note20u现在是oneui5.1

**Comment by lisonge** [comment]
https://github.com/gkd-kit/gkd/issues/115#issuecomment-1751026613
